src/howard/howard/templates/marion/certificates/realisation.html

```diff
@@ -43,7 +43,7 @@ <h1>CERTIFICAT DE REALISATION</h1>
         </section>
         <section id="student">
           <div class="name">
-            <span class="gender">{{ student.gender }}</span>
+            <span class="gender">{{ student.gender.value }}</span>
```

Comment:
See Enum objects suggestion:

```suggestion
            <span class="gender">{{ student.gender }}</span>
```